src/data/textNotification.ts

```diff
@@ -1,3 +1,15 @@
 
-    export const createProduct = 'Product was successfully created';
-    export const deleteProduct = 'Product was successfully deleted'
+export const createProduct = 'Product was successfully created';
+export const deleteProduct = 'Product was successfully deleted';
+
+export enum INPUTFIELD {
+    NAME = 'Name',
+    PRICE = 'Price',
+    AMOUNT = 'Amount',
+};
+
+export const validationMessage: {'Name': string, 'Price': string, 'Amount': string} = {
```

Comment:
Пробуй переиспользовать созданьіе сущности

```suggestion
export const validationMessage: Record<
  (typeof INPUTFIELD)[keyof typeof INPUTFIELD],
  string
> = {
```